sdk/assembly/__tests__/runtime.spec.ts

```diff
@@ -770,12 +785,12 @@ describe("promise batches", () => {
       .delete_account("bene.account");
   });
 
-  it("should support contract batch transactions", () => {
-    VMContext.setPrepaid_gas(10000000000000);
-    expect(Context.accountBalance).toBe(u128.from(9));
-    ContractPromiseBatch.create("alice").transfer(u128.from(1));
-    expect(Context.accountBalance).toBe(u128.from(8)); // this number is surprising
-  });
+  // it("should support contract batch transactions", () => {
```

Comment:
@willemneal why are this and other tests commented out?

Reply:
They should be deleted. They are cross contract tests that don't really test anything and should be in unit tests anyway.